src/PluginRegistrationDialog.cpp

```diff
@@ -2,58 +2,30 @@
 
   Audacity: A Digital Audio Editor
 
-  PluginManager.cpp
+  PluginRegistrationDialog.cpp
 
-  Leland Lucius
+  Paul Licameli split from PluginManager.cpp
```

Comment:
Again, some additions near the top of this new file to make it compile.  Otherwise mostly you see in red what was not copied from PluginManager.cpp; what is not read is easily seen to be preserved unchanged.